src/ember-app/app/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
Here's one of the big problems with using ember cli and single-spa together -- ember cli assumes it has control over the html file, but in order to fit into a single-spa architecture, you have to share the html page with other apps. See #33 for more discussion